src/java/voldemort/server/VoldemortConfig.java

```diff
@@ -355,6 +356,8 @@ public VoldemortConfig(Props props) {
                                                      VoldemortConfig.DEFAULT_KERBEROS_REALM);
         this.fileFetcherClass = props.getString("file.fetcher.class",
                                                 VoldemortConfig.DEFAULT_FILE_FETCHER_CLASS);
+        this.readOnlyStatsFileEnabled = props.getBoolean("readonly.stats.file.enabled", true);
+        this.readOnlyMaxVersionsStatsFile = props.getInt("readonly.stats.file.max.versions", 50);
```

Comment:
I think we can have a much bigger default here... These files should be very small, so I don't think there's a lot of value in aggressively pruning them. How about making the default 1000? For stores that push every 4 hours, 1000 pushes will yield less than 6 months of stats. For stores that push once a day, it's basically going to last "forever". That sounds reasonable to me.

We can always bump it back down if we have stats disk footprint problems. What do you think?